test/integration/build-output/test/index.test.js

```diff
@@ -129,7 +129,7 @@ describe('Build Output', () => {
           expect(parseFloat(err404Size)).toBeCloseTo(gz ? 3.06 : 8.15, 1)
           expect(err404Size.endsWith('kB')).toBe(true)
 
-          expect(parseFloat(err404FirstLoad)).toBeCloseTo(gz ? 66.4 : 203, 1)
+          expect(parseFloat(err404FirstLoad)).toBeCloseTo(gz ? 66.5 : 203, 1)
```

Comment:
Had to bump the expected size by 100 bytes for the tests to pass. Could be to do with an extra prefixed css rule that was caused by the new version of `caniuse-lite`. Just commenting on this to bring the reviewers’ the attention 👀